sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs

```diff
@@ -99,6 +99,33 @@ protected VirtualMachineScaleSetExtension GetAzureDiskEncryptionExtension()
             return vmExtension;
         }
 
+        protected VirtualMachineScaleSet CreateDefaultVMScaleSetInput_Flex(
+            string rgName,
```

Comment:
Why is there no re-recorded json file for this file?

Reply:
This is not an added test, it is a function I added to create a flex VMSS. I don't see any previous json files associated to this file